.github/workflows/ci.yml

```diff
@@ -0,0 +1,18 @@
+name: CI
+
+on:
+  push:
```

Comment:
Až bude přepis hotový, tak by se tohle přehodilo na `master`.

Reply:
Proč to limitovat na branch? Kontroly by měly fungovat pro všechny PR. Tím, že necháme specifikaci pouze na nekonkrétní PR, tak pak se nám to postupně zprovozní na všech větví:
1. V této
2. Rework
3. Master

Reply:
Kdybysme to momentálně neomezovali a mergli tenhle PR, tak nám to bude ty kontroly pouštět i na `master`, kde neprojdou a bude nám chodit zbytečný spam?

Reply:
Přemýšlím, kde jsme se myšlenkově minuli – všiml sis, že tohle je PR na merge do masteru? Kdybysme to mergnuli jen do `rework`, bude ta action fungovat? Myslel jsem, že GitHub Actions frčí jen z výchozí větve repa – [tady je k tomu nějaká diskuze](https://github.community/t5/GitHub-Actions/Workflow-files-only-picked-up-from-master/td-p/30348), ale nevím, jak to dopadlo.

Reply:
A pardon, zapomněl jsem zmínit změnu cílové větve na `rework`. Nicméně ta limitace je pro mne novinka. Dává to smysl, ale mám pocit, že jsem se nesetkal s tím, že by to takto fungovalo, resp. u posledních GH actions jsem je testoval přímo na dané branch.

Reply:
Přehodil jsem cílovou větev na rework a smazal to omezení na větve, můžem zkusit mergnout do `rework` a uvidíme, jestli to bude fungovat – spíš asi jo.